analyzers/src/SonarAnalyzer.CSharp/Rules/BooleanLiteralUnnecessaryCodeFix.cs

```diff
@@ -223,10 +223,28 @@ private static SyntaxNode FindNodeToKeep(BinaryExpressionSyntax binary)
 
             #endregion
 
+            #region ==/!= one side boolean
```

Comment:
The QG does not like this, but I think it's nice since we use it twice above.

Reply:
I think we should replace all the regions with comments to respect our coding style.